### PR TITLE
feat(console): edit standard connector

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

feature: edit standard connector data.

## Testing

截屏2022-12-01 下午4 30 49

截屏2022-12-01 下午4 31 10

### CODE REVIEW

File: packages/console/src/pages/Connectors/components/ConnectorForm/index.tsx [hunk 1]
```diff
@@ -43,28 +43,27 @@ const Form = ({ connector }: Props) => {
...
-          )}
-          {darkVisible && (
+            </>
+          ) : (
             <Button
               size="small"
               type="text"
-              title="connectors.guide.logo_dark_collapse"
+              title="connectors.guide.logo_dark_show"
               onClick={toggleDarkVisible}
```
Comment (charIeszhao) [inline packages/console/src/pages/Connectors/components/ConnectorForm/index.tsx:67]: nit: These two buttons are almost identical, should just dynamically render the titles

File: packages/core/src/routes/connector.ts [hunk 2]
```diff
@@ -193,10 +193,9 @@ export default function connectorRoutes<T extends AuthedRouter>(router: T) {
         validateConfig(config);
       }
 
-      // FIXME @Darcy [LOG-4696]: revisit databaseMetadata check when implementing AC
-
-      const connector = await updateConnector({ set: body, where: { id }, jsonbMode: 'replace' });
-      ctx.body = { ...connector, metadata, type };
+      await updateConnector({ set: body, where: { id }, jsonbMode: 'replace' });
+      const connector = await getLogtoConnectorById(id);
+      ctx.body = transpileLogtoConnector(connector);
```
Comment (xiaoyijun) [inline packages/core/src/routes/connector.ts:198]: Nit(Not realted to this PR):
I found the `POST /connectors` handler does not return the correct connector data type. Maybe add an issue to track this or update it later.

Comment (xiaoyijun) [reply packages/core/src/routes/connector.ts:198]: image

**Review by xiaoyijun — APPROVED** [review]
LGTM

File: packages/core/src/routes/connector.ts (hunk 2 above)
Comment (wangsijie) [reply packages/core/src/routes/connector.ts:198]: LOG-4857

File: packages/console/src/pages/Connectors/components/ConnectorForm/index.tsx (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/pages/Connectors/components/ConnectorForm/index.tsx]: updated

**Pull request merged**